egs/sre08/v1/sid/nnet3/xvector/extract_xvectors.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+# Copyright     2017  David Snyder
+#               2017  Johns Hopkins University (Author: Daniel Povey)
+#               2017  Johns Hopkins University (Author: Daniel Garcia Romero)
+# Apache 2.0.
+
+# This script extracts embeddings (called "xvectors" here) from a set of
+# utterances, given features and a trained DNN.
```

Comment:
it would be great if you could summarize how it differs from the regular extract_ivectors.sh script